File: 프로그래머스/Level1/수박수박수박수박수박수?.py
# https://programmers.co.kr/learn/courses/30/lessons/12922

def solution(n):
    answer = ''
    for i in range(1, n + 1):
        if i % 2 == 0:
            answer += '박'
        else:
            answer += '수'
    return answer

File: 프로그래머스/Level1/수박수박수박수박수박수?.js
// https://programmers.co.kr/learn/courses/30/lessons/12922

const solution = (n) => {
	let answer = '';
	for (let i = 1; i <= n; i++) {
		i % 2 === 0 ? (answer += '박') : (answer += '수');
	}
	return answer;
};
